### PR TITLE
render: meta: display rule paths on separate lines

### LINKED ISSUE #971: when there are multiple rule paths, render each on its own line
Screen Shot 2022-04-06 at 1 12 33 PM

### PR DESCRIPTION
**Pull request by williballenthin**
closes #971

image

[x] No CHANGELOG update needed

### CODE REVIEW

File: capa/render/verbose.py [hunk 1]
```diff
@@ -71,6 +71,20 @@ def render_meta(ostream, doc):
...
+    #
+    #     base address            0x10000000
+    #     rules                   /path/1
+    #                             /path/2
+    #     function count          2
+    if len(doc["meta"]["analysis"]["rules"]) > 1:
+        idx = rows.index(("rules", doc["meta"]["analysis"]["rules"][0])) + 1
+        for rule in doc["meta"]["analysis"]["rules"][1:]:
+            rows.insert(idx, ("", rule))
+            idx += 1
```
Comment (mr-tz) [inline capa/render/verbose.py:86]: "\n".join(...) should also work?

```
print(tabulate.tabulate([("left column", "right 1\nright 2")]))
-----------  -------
left column  right 1
             right 2
-----------  -------
```

Comment (williballenthin) [reply capa/render/verbose.py:86]: whoa, thats a lot cleaner, thanks!

**Review by mr-tz — APPROVED** [review]
see comment, but the proposal is also fine

File: capa/render/verbose.py (hunk 1 above)
Comment (williballenthin) [reply capa/render/verbose.py]: fixed in fccca823c5746cebedea240e3833bc0ee72d988d

Comment (mr-tz) [reply capa/render/verbose.py]: oh yeah!

**Pull request merged**